pkg/infrastructure/azure/azure.go

```diff
@@ -255,9 +252,8 @@ func (p *Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput
 		logrus.Debugf("StorageAccount.ID=%s", *storageAccount.ID)
 	}
 
-	// Upload the image to the container
-	_, skipImageUpload := os.LookupEnv("OPENSHIFT_INSTALL_SKIP_IMAGE_UPLOAD")
-	if !(skipImageUpload || platform.CloudName == aztypes.StackCloud) {
```

Comment:
:bulb: (cleanup): There is another reference of the variable to clean up too.

https://github.com/openshift/installer/blob/5a9edbbf5aaf3d79fd7f5ddcfe2ee6bd9f8fd45e/pkg/asset/installconfig/azure/validation.go#L691-L699